include/highfive/H5Object.hpp

```diff
@@ -32,6 +32,16 @@ enum class ObjectType {
     Other  // Internal/custom object type
 };
 
+///
+/// \brief The possible types of group entries (link concept)
+///
+enum class LinkType {
+    Hard,
+    Soft,
+    External,
+    Other  // Reserved or User-defined
+};
+
```

Comment:
Is there a good reason to move "LinkType" to another header? Please don't introduce breaking changes, unless it's last resort.

Reply:
I think for now it can reside in its usual header. I will try to move it back.